tpl/strings/strings_test.go

```diff
@@ -35,12 +35,20 @@ func TestChomp(t *testing.T) {
 		s      interface{}
 		expect interface{}
 	}{
-		{"\n a\n", template.HTML("\n a")},
-		{"\n a\n\n", template.HTML("\n a")},
-		{"\n a\r\n", template.HTML("\n a")},
-		{"\n a\n\r\n", template.HTML("\n a")},
-		{"\n a\r\r", template.HTML("\n a")},
-		{"\n a\r", template.HTML("\n a")},
+		// string to string
+		{"\n a\n", "\n a"},
+		{"\n a\n\n", "\n a"},
+		{"\n a\r\n", "\n a"},
+		{"\n a\n\r\n", "\n a"},
+		{"\n a\r\r", "\n a"},
+		{"\n a\r", "\n a"},
+		// html to html
+		{template.HTML("\n a\n"), template.HTML("\n a")},
+		{template.HTML("\n a\n\n"), template.HTML("\n a")},
+		{template.HTML("\n a\r\n"), template.HTML("\n a")},
+		{template.HTML("\n a\n\r\n"), template.HTML("\n a")},
+		{template.HTML("\n a\r\r"), template.HTML("\n a")},
+		{template.HTML("\n a\r"), template.HTML("\n a")},
```

Comment:
I would rather see the test cases be strings and the test check both types per case.

Reply:
Sounds reasonable, updated